bedrock/base/templates/base-new.html

```diff
@@ -0,0 +1,149 @@
+{% from "macros.html" import share_cta with context %}
```

Comment:
New pages in the upcoming redesign should be totally divorced from sandstone and for now creating a new base template was a start down that path (and I wanted to get rid of the outer-wrapper and wrapper divs). We need to formulate a better plan for the redesign. This base template should be considered temporary, as hacky as it is, and the "new" in the filename is totally a joke :)